### PR TITLE
update explorer docs with granular configuration

### PR DESCRIPTION
**Pull request by jpellizzari**
Related to https://github.com/weaveworks/weave-gitops-enterprise/pull/3485

### CODE REVIEW

**Review by opudrovs — APPROVED** [review]

File: website/docs/explorer/getting-started.mdx [hunk 1]
```diff
@@ -35,7 +35,20 @@ metadata:
 spec:
   # ... other spec components
   values:
-    enableExplorer: true
+  explorer:
```
Comment (enekofb) [inline website/docs/explorer/getting-started.mdx:38]: given its a getting started page, a minimal configuration would be better 

```
explorer:
  enabled: true # global enable/disable flag
```
and then leave other cases for the user to go onto the configuration page 

https://docs.gitops.weave.works/docs/explorer/configuration/

where to give a little more guidance and the configuration options that we provide 

https://docs.gitops.weave.works/docs/explorer/configuration/#setup

File: website/docs/explorer/getting-started.mdx [hunk 2]
```diff
@@ -35,7 +35,20 @@ metadata:
...
+      serviceAccount:
+        name: "collector"
+        namespace: "flux-system"
+    cleaner:
+      disabled: false
+    enabledFor:
+      - applications
+      - sources
+      - gitopssets
+      - templates
```
Comment (enekofb) [inline website/docs/explorer/getting-started.mdx:51]: it feels that we should expand with comments to provide guidance to the user on how use these flags. for example: the user could think that is configuration for deciding which type of data to collect and not for the UI only

**Review by enekofb — COMMENTED** [review]
Added comment with suggestion on how to split the configuration info.

**Review by enekofb — APPROVED** [review]
approving and merging to get it in today's wego release

followup around explorer docs and getting started vs configuration https://docs.gitops.weave.works/docs/explorer/configuration/

**Pull request merged**